resources/task.rb

```diff
@@ -43,6 +43,7 @@
 attribute :start_day, kind_of: String, default: nil
 attribute :start_time, kind_of: String, default: nil
 attribute :day, kind_of: [String, Integer], default: nil
+attribute :months, kind_of: String, default: nil
```

Comment:
actually looking at it from the gui side task scheduler, with the weekly and monthly settings you can specify multiple days. Weekly gives you mon,tue,wed,etc... options and monthly days gives you the 1-31 etc... options